appcues/src/main/java/com/appcues/debugger/DebuggerViewModel.kt

```diff
@@ -188,7 +202,7 @@ internal class DebuggerViewModel(
             is Idle -> {
                 when (state.mode) {
                     is Debugger -> _uiState.value = Expanded(state.mode)
-                    is ScreenCapture -> Unit
+                    is ScreenCapture -> AppcuesActivityMonitor.activity?.captureScreen(config, contextResources)
```

Comment:
note, relying on the new `AppuesActivityMonitor` at this point. If this bubbled up to the `AppcuesDebuggerManager` we'd already have a `currentActivity` reference there.